src/main/java/com/vdovich/tbfm/GreetingController.java

```diff
@@ -0,0 +1,17 @@
+package com.vdovich.tbfm;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+@RequestMapping
```

Comment:
Wrong annotation. Please, change to a proper annotation and find out the difference between them.